app/assets/translations/appConstants/bebbo/RSen.ts

```diff
@@ -1,453 +1,458 @@
+                "developScreentipsText": "Tips from Doctors",
+                "developScreenmileStone": "Milestone",
+                "developScreenrelatedArticleText": "View related advice",
+                "developScreenrelatedAct": "Related games ",
+                "developScreenviewDetails": "View details",
+                "articleScreenheaderTitle": "Advice ",
+                "articleScreensearchPlaceHolder": "Search for keywords",
+                "articleScreennoSearchResult": "No advice found",
+                "articleScreenshareText": "Share",
+                "articleScreenaddtoFavText": "Add to favourites",
+                "growthScreenheaderTitle": "Child's growth",
+                "growthScreennoGrowthData": "No growth measurements recorded",
+                "growthScreennoGrowthDataHelpText": "This will help you track your child's growth",
+                "growthScreenaddNewBtntxt": "Add new measurement",
+                "growthScreeneditNewBtntxt": "Edit growth measurement",
+                "growthScreenweightForHeight": "Weight for height",
+                "growthScreenheightForAge": "Height for age",
+                "growthScreensubHeading": "Growth measurement",
+                "growthScreenprematureText": "Premature",
+                "growthScreenlastMeasureText": "Last measured on {{measureDate}}",
+                "growthScreenallMeasurements": "All measurements",
+                "growthScreenhText": "Height",
+                "growthScreenwText": "Weight",
+                "growthScreeneditText": "Edit",
+                "growthScreendelText": "Delete",
+                "growthScreenkgText": "kg",
+                "growthScreencmText": "cm",
+                "growthScreensumHeading": "Summary",
+                "growthScreenallMeasureHeader": "All measurements",
+                "growthScreendeletebtnText": "Delete",
+                "growthScreendateMeasurementText": "Date of measurement",
+                "growthScreenenterDateMeasurementText": "Enter date of measurement",
+                "growthScreenwhereMeasured": "Where was the child measured?",
+                "growthScreendoctorMeasurePlace": "At the doctor's",
+                "growthScreenhomeMeasurePlace": "At home",
+                "growthScreenenterMeasuresText": "Enter weight and height",
+                "growthScreenenterDoctorRemarkText": "Doctor's remark or comment",
+                "growthScreenenterDoctorRemarkTextPlaceHolder": "Enter your doctor's remark or comment",
+                "growthScreensaveMeasures": "Save",
+                "growthScreensaveMeasuresDetails": "Save",
+                "growthScreenaddWeight": "Add weight",
+                "growthScreenaddHeight": "Add height",
+                "growthScreennewGrowthBottomText": "Measuring a child at home is less accurate than measuring at the doctor's and can affect the accuracy of the growth chart.",
+                "growthScreenrelatedArticle": "Related advice ",
```

Comment:
Consider breaking this into two sentences for clarity: 
"Measuring a child at home is less accurate than measuring at the doctor's. This can affect the accuracy of the growth chart."

Reply:
We can update this in string changes. kindly create one task for all string changes.